### PR TITLE
Fixing static analysis vulnerabilities for aim

### PR DESCRIPTION
**Pull request by sbagalur**
Fixing the static analysis vulnerabilities that were reported through bandit runs/ marking the false positives.

### CODE REVIEW

**Review by tbachman — APPROVED** [review]

**Review by christides11 — APPROVED** [review]

File: aim/agent/aid/universes/aci/aci_universe.py [hunk 1]
```diff
@@ -519,7 +520,9 @@ def _do_login_loop(self):
             LOG.error(
                 'Could not refresh APIC login due to ApicResponseNotOk')
         # All others - just login after timeout
-        except Exception:
+        except Exception as e:
+            LOG.info("Exception occurred. Log in after timeout. error: %s",
+                     str(e))
             pass
```
Comment (tbachman) [inline aim/agent/aid/universes/aci/aci_universe.py:526]: I think you can get rid off the pass statement here.

File: aim/agent/aid/universes/aci/tenant.py [hunk 2]
```diff
@@ -451,15 +451,19 @@ def _main_loop(self):
             LOG.info("Scheduled tree reset for root %s" % self.tenant_name)
             try:
                 self._unsubscribe_tenant()
-            except Exception:
+            except Exception as e:
+                LOG.info("Exception has occurred while unsubscribing tenant %s"
+                         ", error: %s" % (self.tenant_name, str(e)))
                 pass
```
Comment (tbachman) [inline aim/agent/aid/universes/aci/tenant.py:457]: same for here and the one below

File: aim/db/migration/data_migration/status_add_dn.py [hunk 3]
```diff
@@ -60,6 +63,8 @@ def migrate(session):
                 res = store.make_resource(res_class, db_res)
                 session.execute(update(Status).where(
                     Status.c.id == st.id).values(resource_dn=res.dn))
-            except Exception:
+            except Exception as e:
                 # Silently ignore
+                LOG.info("Exception occurred while migrating status."
+                         "error: %s", str(e))
```
Comment (tbachman) [inline aim/db/migration/data_migration/status_add_dn.py:69]: I think you can get rid of the pass statement here.

**Review by tbachman — CHANGES_REQUESTED** [review]
Some minor changes.

**Review by tbachman — APPROVED** [review]

**Pull request merged**